src/tox/tox_env/python/virtual_env/package/pyproject.py

```diff
@@ -312,8 +319,10 @@ def _ensure_meta_present(self, for_env: EnvConfigSet) -> None:
         self.setup()
         end = self._frontend
         if for_env["package"] == "editable":
+            self._setup_build_requires("editable")
```

Comment:
There's a reason we did not have setup calls here, because I feel doesn't belong here 🤔